src/main/java/org/javarosa/core/model/FormDef.java

```diff
@@ -1455,41 +1450,6 @@ public void initialize(boolean newInstance, boolean isCompletedInstance, Instanc
         }
     }
 
-    private void tryAttachingRemoteInstances(RemoteInstanceFetcher remoteInstanceFetcher) throws RemoteInstanceFetcher.RemoteInstanceException {
-        ArrayList<DataInstance> replacedInstances = new ArrayList<>();
-        Enumeration<DataInstance> instances = getNonMainInstances();
-        while (instances.hasMoreElements()) {
-            DataInstance instance = instances.nextElement();
-            if (instance instanceof ExternalDataInstance &&
-                    instance.getRoot() == null &&
-                    ((ExternalDataInstance)instance).getRemoteUrl() != null) {
-                try {
-                    ExternalDataInstance externalDataInstance = (ExternalDataInstance)instance;
-                    ExternalDataInstance newExternalDataInstance = remoteInstanceFetcher.getRemoteDataInstance(instance.getInstanceId(),
-                            externalDataInstance.useCaseTemplate(),
-                            new URI(externalDataInstance.getRemoteUrl()));
-                    if (newExternalDataInstance != null && newExternalDataInstance.getRoot() != null) {
-                        replacedInstances.add(newExternalDataInstance);
-                    }
-                } catch (UnfullfilledRequirementsException | XmlPullParserException |
-                        InvalidStructureException | IOException | URISyntaxException e) {
-                    String errorMessage;
-                    if (e instanceof URISyntaxException) {
-                        errorMessage = "Invalid url found for the remote instance " + instance.getName() + ".";
-                    } else if (e instanceof IOException) {
```

Comment:
Should we retain this error handling elsewhere aka try to give different error messages on invalid data vs a network error ?